cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -63,7 +63,9 @@ spec:
       containers:
         - name: manager
           args:
-            - --port=8443
+            - --port=HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_PORT
+            - --health-addr=:HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_HEALTH_PORT
+            - --prometheus-port=HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_PROMETHEUS_PORT
```

Comment:
thoughts about calling this `metrics-port`? since the exporter can target more than Prometheus

Reply:
Sounds reasonable, i will update PR when i get to work tomorrow or later tonight.

Reply:
I guess the main reason why I used prometheusPort is due to that the flag is `--prometheus-port` but if you change this flag in the future to make it more generic we won't have to change the helm values at least :)
I will just verify the change locally and then I will update the PR.